### PR TITLE
Corda-725 - Change AMQP identifier

### PR DESCRIPTION
**Pull request by fenryka**
Move from placeholder to assigned value

### CODE REVIEW

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -63,6 +63,8 @@ UNRELEASED
 * A new function ``checkCommandVisibility(publicKey: PublicKey)`` has been added to ``FilteredTransaction`` to check
   if every command that a signer should receive (e.g. an Oracle) is indeed visible.
 
+* Change the AMQP serialiser to use the oficially assigned R3 identifier rather than a placeholder
```
Comment (andr3ej) [inline docs/source/changelog.rst]: other entries end with dot

File: node-api/src/main/kotlin/net/corda/nodeapi/internal/serialization/amqp/Schema.kt [hunk 2]
```diff
@@ -18,8 +18,15 @@ import java.util.*
 import net.corda.nodeapi.internal.serialization.carpenter.Field as CarpenterField
 import net.corda.nodeapi.internal.serialization.carpenter.Schema as CarpenterSchema
 
-// TODO: get an assigned number as per AMQP spec
-const val DESCRIPTOR_TOP_32BITS: Long = 0xc0da0000
+/**
+ * R3 AMQP assigned enterprise number
+ *
+ * see [https://www.iana.org/assignments/enterprise-numbers/enterprise-numbers]
```
Comment (andr3ej) [inline node-api/src/main/kotlin/net/corda/nodeapi/internal/serialization/amqp/Schema.kt]: kdoc square brackets aren't applicable to urls, resulting in garbage apidocs output. helpfully, i don't know what does work

**Comment by fenryka** [comment]
@andr3ej done!

File: node-api/src/test/kotlin/net/corda/nodeapi/internal/serialization/amqp/EvolvabilityTests.kt [hunk 3]
```diff
@@ -30,6 +30,7 @@ class EvolvabilityTests {
         // data class C (val a: Int, val b: Int)
         // val sc = SerializationOutput(sf).serialize(C(A, B))
         // f.writeBytes(sc.bytes)
+        // println (path)
```
Comment (andr3ej) [inline node-api/src/test/kotlin/net/corda/nodeapi/internal/serialization/amqp/EvolvabilityTests.kt]: not specific to this PR
* you can probably write directly to the correct resources directory by using the ProjectStructure hack or something similar
* and/or i would add a little script to minimise the manual steps in regenerating the binaries, and a gradle task to launch it

Comment (fenryka) [reply node-api/src/test/kotlin/net/corda/nodeapi/internal/serialization/amqp/EvolvabilityTests.kt]: Yeah, but that's all "future work" :) For now it works and if I get super annoyed at doing this manually I'll sort it out at that point

Although hopefully they won't need regenerating

File: node-api/src/main/kotlin/net/corda/nodeapi/internal/serialization/amqp/Schema.kt [hunk 4]
```diff
@@ -21,7 +21,7 @@ import net.corda.nodeapi.internal.serialization.carpenter.Schema as CarpenterSch
 /**
  * R3 AMQP assigned enterprise number
  *
- * see [https://www.iana.org/assignments/enterprise-numbers/enterprise-numbers]
+ * see [here]([https://www.iana.org/assignments/enterprise-numbers/enterprise-numbers)
```
Comment (andr3ej) [inline node-api/src/main/kotlin/net/corda/nodeapi/internal/serialization/amqp/Schema.kt]: brackets just need balancing

Comment (fenryka) [reply node-api/src/main/kotlin/net/corda/nodeapi/internal/serialization/amqp/Schema.kt]: Done!!!

**Pull request merged**